lib/solidus_support/engine_extensions.rb

```diff
@@ -110,7 +110,7 @@ def enable_solidus_engine_support(engine)
           decorators_path = root.join("lib/decorators/#{engine}")
           controllers_path = root.join("lib/controllers/#{engine}")
           config.autoload_paths += decorators_path.glob('*')
-          config.autoload_paths += controllers_path.glob('*')
+          config.autoload_paths << controllers_path if controller_path.exist?
```

Comment:
This works, but should the line 112 (just above) not receive the same treatment?

Reply:
Good call, I'd expect that folder to have an additional layer and be similar to `app/` in that regard, eg. `lib/decorators/solidus_foo/models/spree/product/my_decorator.rb`